Assets/Scripts/UI/AbilityCooldown.cs

```diff
@@ -0,0 +1,58 @@
+using Cysharp.Threading.Tasks;
+using System;
+using System.Threading;
+using UnityEngine;
+using TMPro;
+
+namespace RogueApeStudio.Crusader.UI.Cooldown
+{
+    public class AbilityCooldown : MonoBehaviour
+    {
+        private CancellationTokenSource _cancellationTokenSource;
```

Comment:
Incorrect order, should be:
`Serialized Fields`
`Private Fields`

Reply:
I've commented on it when I've been attached to the a PR, hence the reason I've asked to be added to all PR's.